kobo/apps/openrosa/apps/api/permissions.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, *args, **kwargs):
 
         # `PATCH` should already be set properly by DRF, but it used to be
         # explicitly assigned here as well. Double-check that it's right
-        assert self.perms_map['PATCH'] == ['%(app_label)s.change_%(model_name)s']
+        # assert self.perms_map['PATCH'] == ['%(app_label)s.change_%(model_name)s']
```

Comment:
I guess it is because of debugging but we should keep the assertion. 
Let's remove the `#`